contracts/src/ERC20.sol

```diff
@@ -0,0 +1,235 @@
+// SPDX-License-Identifier: MIT
+// SPDX-FileCopyrightText: 2023 Axelar Network
```

Comment:
i think remove this is we've made significant changes?

Reply:
Only the license identifier is in the original files, so that's all we need to keep

Reply:
The MIT license, like most permissive free software licenses, requires that the original copyright be maintained in derived works:

> The above copyright notice and this permission notice shall be included in all
> copies or substantial portions of the Software.

So rather add a second `SPDX-FileCopyrightText` line with our copyright.